### PR TITLE
Changing layer5 link to meshery link

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Changing the layer5 link to meshery link

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sayantan1413** [comment]
@kumarabd Can you verify once?

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ assignees: ''
 
 ---
 **Contributor Resources**
-- [Meshery documentation site](https://meshery.layer5.io/docs/)
-- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
-- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
+- [Meshery documentation site](https://docs.meshery.io/)
```
Comment (kumarabd) [inline .github/ISSUE_TEMPLATE/documentation.md:16]: @sayantan1413 lets not change this

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ This PR fixes #
 **Notes for Reviewers**
 
 
-**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
+**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
```
Comment (kumarabd) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: lets not change this

File: .github/ISSUE_TEMPLATE/documentation.md (hunk 1 above)
Comment (sayantan1413) [reply .github/ISSUE_TEMPLATE/documentation.md:16]: Okay so @kumarabd I will not change the links for the meshery documentation right?

Comment (kumarabd) [reply .github/ISSUE_TEMPLATE/documentation.md:16]: yes, just the GitHub endpoint needs to be updated.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (sayantan1413) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Okay

**Comment by sayantan1413** [comment]
@kumarabd I have made the changes can you verify it once?

**Pull request merged**